packages/labs/react/src/create-component.ts

```diff
@@ -255,6 +259,8 @@ export const createComponent = <I extends HTMLElement, E extends Events>(
       // iterate again when setting properties.
       this._elementProps = {};
       for (const [k, v] of Object.entries(this.props)) {
+        if (propertySieve.has(k)) continue;
```

Comment:
I wouldn't generify this yet. A `=== '__forwardedRef'` check is going to be much faster than a `has()` check. We can change later, and for lest than 7-10 keys an array in check will be faster than a Set too.

Reply:
sounds good to me!